deployment/makefiles/scripts/configurations.json

```diff
@@ -0,0 +1,88 @@
+[
+  {
+    "Classification": "spark",
+    "Properties": {
+      "maximizeResourceAllocation": "true"
+    }
+  },
+  {
+    "Classification": "spark-defaults",
+    "Properties": {
+        "spark.driver.maxResultSize": "3G",
+        "spark.dynamicAllocation.enabled": "true",
+        "spark.shuffle.service.enabled": "true",
+        "spark.shuffle.compress": "true",
+        "spark.shuffle.spill.compress": "true",
+        "spark.rdd.compress": "true",
+        "spark.yarn.executor.memoryOverhead": "1G",
```

Comment:
FWIW, I've needed to use 2G when doing planet-scale processing, but I'm also not using any GC options.

(Is this intended to provide defaults, or will these always be provided in the EMR jobs triggered via the `Makefile`?)

Reply:
Noted.  (As I understand it, these are cluster-wide defaults.)